### ISSUE TITLE
Error: Cannot find module './generated/nexus-prisma'

### ISSUE THREAD

**Issue by deadcoder0904** [graphql-nexus/nexus-plugin-prisma#363]
I am trying to use Nexus with Prisma 1 and it gives this error when I try to start `./src/index.js`.

```bash
yarn run v1.17.3
$ node src/index.js
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module './generated/nexus-prisma'
Require stack:
- ~/prisma1-server/src/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (~/prisma1-server/src/index.js:6:23)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '~/prisma1-server/src/index.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I have uploaded the code here → https://github.com/deadcoder0904/graphql-and-urql-by-example/tree/master/prisma1-server

I have the generated files so I don't know why it's not working. For additional context, here's `prisma.yml` ⸺

```yaml
# Specifies the HTTP endpoint of your Prisma API.
endpoint: 'http://localhost:4466'

# Defines your models, each model is mapped to the database as a table.
datamodel: datamodel.prisma

# Specifies the language and directory for the generated Prisma client.
generate:
  - generator: javascript-client
    output: ../src/generated/prisma-client/
  - generator: graphql-schema
    output: ../src/generated/schema.graphql
hooks:
  post-deploy:
    - prisma generate
    - npx nexus-prisma-generate --client ./src/generated/prisma-client --output ./src/generated/nexus-prisma

# Seeds initial data into the database by running a script.
seed:
  run: node ./prisma/seed.js
```

**Comment by deadcoder0904** [comment]
This is weird. So I was facing the same issue with Prisma 2 now that `nexus-prisma` is no longer a generator.

This is a TS example but I'm doing JS → https://github.com/prisma-labs/nexus-prisma/blob/master/examples/blog/package.json

So I didn't know how to generate `nexus-prisma`. While writing this draft, I found the answer so I hope it helps someone.

What I changed was removed `generator nexus_prisma` code from `schema.prisma` 👇

```graphql
generator nexus_prisma {
  provider = "nexus-prisma"
}
```

Then I changed the import from

```js
const {nexusPrismaPlugin} = require('@generated/nexus-prisma');
```

to

```js
const {nexusPrismaPlugin} = require('nexus-prisma');
```

And installed `nexus-prisma` by doing

```bash
$ yarn add nexus-prisma
```

This made it work. Hope it helps :)
